oci/private/image.sh

```diff
@@ -1,144 +1,174 @@
 #!/usr/bin/env bash
 set -o pipefail -o errexit -o nounset
 
+# Add hermetically built jq, regctl, coreutils into PATH.
+PATH="{{jq_path}}:$PATH"
```

Comment:
is there a reason to put them on PATH rather than call them using absolute paths? Is it to ensure that subprocesses see them as well?